src/domains/plugin/components/Comments/components/TimeAgo/TimeAgo.tsx

```diff
@@ -0,0 +1,31 @@
+import { DateTime } from "@arkecosystem/platform-sdk-intl";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const dateDifference = (date: string) => {
+	const now = DateTime.make();
+	const target = DateTime.make(date);
+
+	for (const period of ["Years", "Months", "Days", "Hours", "Minutes"]) {
+		// @ts-ignore
+		const count: number = now[`diffIn${period}`](target);
+
+		if (count > 0) {
+			return { count, key: period.toUpperCase() };
+		}
+	}
+
+	return { key: "FEW_SECONDS" };
+};
+
+export const TimeAgo = ({ date }: { date: string }) => {
+	const { t } = useTranslation();
+
+	const { count, key } = dateDifference(date);
+
+	return (
+		<span data-testid="timeago">
+			{count ? t(`COMMON.DATETIME.${key}_AGO`, { count }) : t(`COMMON.DATETIME.${key}_AGO`)}
```

Comment:
Could this simply be `return <span data-testid="timeago">{t(`COMMON.DATETIME.${key}_AGO`, { count })}</span>;` and the i18n lib ignores the parameter if it isn't used?

Reply:
that works indeed